charts/gatekeeper-artifacts/values.yaml

```diff
@@ -0,0 +1,2 @@
+installCRDs: false
```

Comment:
never use such constructs, as it implies running the stack twice with in between manipulation...we are aiming to deliver an idempotent desired state.

Reply:
other charts have been using this for helm2 backwards compatibility.

Reply:
Yeah, I know. But this chart is adopted and should be adhering to our requirements.